main/nav_main_offset.py

```
@@ -163,20 +164,26 @@ def main() -> None:
-    if results_root_str is None:
-        results_root_str = os.getenv('NAV_RESULTS_ROOT')
-    if results_root_str is None:
-        raise ValueError('One of configuration variable "results_root" or '
-                         'NAV_RESULTS_ROOT environment variable must be set')
-    results_root = FileCache('nav_results').new_path(results_root_str)
+    if nav_results_root_str is None:
+        nav_results_root_str = os.getenv('NAV_RESULTS_ROOT')
+    if nav_results_root_str is None:
+        raise ValueError('One of --nav-results-root, the configuration variable '
+                         '"nav_results_root" or the NAV_RESULTS_ROOT environment variable must be '
+                         'set')
```

Ruff (0.14.5)

Avoid specifying long messages outside the exception class

(TRY003)